common/common-framework/src/test/java/cn/iocoder/common/framework/util/DateUtilTest.java

```diff
@@ -0,0 +1,56 @@
+package cn.iocoder.common.framework.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+public class DateUtilTest {
+
+    @Test
+    public void testAddDate() {
+        Assert.assertNull(DateUtil.addDate(0, 0));
+        Assert.assertEquals(new Date(1_778_410_800_000L), DateUtil.addDate(
+                new Date(1_515_585_600_000L), 2, 100));
+    }
+
+    @Test
+    public void testFormat() {
+        Assert.assertEquals("", DateUtil.format(null, null));
+        Assert.assertEquals("2018-01-10:12:00:00", DateUtil.format(
+                new Date(1_515_585_600_000L), "yyyy-MM-dd:HH:mm:ss"));
+    }
+
+    @Test
+    public void testGetDayBegin() {
+        Assert.assertNull(DateUtil.getDayBegin(null));
+        Assert.assertEquals(new Date(1_515_542_400_000L), DateUtil.getDayBegin(
```

Comment:
line > 80 chars (81!)
Maybe break the line after the previous comma which would keep the second argument to assertEquals() all on one line.